src/guiFormSpecMenu.cpp

```diff
@@ -746,7 +743,8 @@ void GUIFormSpecMenu::parseTable(parserData* data,std::string element)
 		spec.ftype = f_Table;
 
 		for (unsigned int i = 0; i < items.size(); ++i) {
-			items[i] = unescape_string(items[i]);
+			items[i] = unescape_string(wide_to_utf8(
+				remove_escapes(utf8_to_wide(items[i]))));
```

Comment:
We might want to add data about a number of characters in escapes (let's say, remove the previous ```n``` characters), and that needs to be consistent so the text has to either be always utf8 or always wide, but not sometimes one and sometimes the other.